mbuild/formats/lammpsdata.py

```diff
@@ -181,33 +196,37 @@ def write_lammpsdata(structure, filename, atom_style='full'):
             sigmas = [atom.sigma for atom in structure.atoms]
             epsilon_dict = dict([(unique_types.index(atom_type)+1,epsilon) for atom_type,epsilon in zip(types,epsilons)])
             sigma_dict = dict([(unique_types.index(atom_type)+1,sigma) for atom_type,sigma in zip(types,sigmas)])
-            data.write('\nPair Coeffs # lj\n\n')
+            data.write('\nPair Coeffs # lj \n\n')
+            data.write('#\tepsilon sigma \t     (kcal/mol, \tAngstrom)\n')
```

Comment:
Would it be helpful to add atomtype comments to the pair coeffs section as well?

Reply:
Sure, I will try to add that tmr morning.